test/cpu_debugger.cpp

```diff
@@ -38,8 +38,14 @@
 using namespace ngraph;
 using namespace std;
 
+// These tests are for DEX mode only.
 TEST(debugger, add_breakpoint)
 {
+    if (std::getenv("NGRAPH_CODEGEN") != nullptr &&
```

Comment:
Can we extract this out into a static method? Gtest probably has a better way to do this.